Assets/Scripts/CharacterController.cs

```diff
@@ -29,18 +33,35 @@ void Start()
         {
             throw new MissingComponentException("Character controller needs a rigidbody");
         }
+        count = 0;
+        SetCountText();
+
+       
     }
 
+
+
     // Update is called once per frame
     void Update()
     {
         CurrentMovementDirection = GetMovementDirectionFromPlayerInputs();
+
+        if (count > 0 & Input.GetKey(Throw))

```

Comment:
Im not sure if you meant to use a bitwise AND operation here, but for the logical AND in C# it's && which may be more suitable for this condition check